### DISCUSSION TITLE
Auto shuffle feature for playlists

### DISCUSSION THREAD

**Discussion by Brandon23031** [jagrosh/MusicBot#900] [Ideas]
I don't know if the bot already has this capability, but it would be nice to have an auto shuffle feature so that it's not always the same song playing first.

Please do let me know if there is already a feature like this.

Thanks in advance!

**Comment by jagrosh** [comment]
https://jmusicbot.com/playlists#-local-playlists
> Lines starting with # or // are ignored for songs. To make a playlist automatically shuffle when loaded, add #shuffle or //shuffle on its own line somewhere in the playlist.

**Reply by Brandon23031** [reply]
Thanks a bunch, appreciate it!
